integration/substrate/ink_cross_calls.spec.ts

```diff
@@ -0,0 +1,44 @@
+import expect from 'expect';
+import { weight, createConnection, deploy, transaction, aliceKeypair, query, } from './index';
+import { ContractPromise } from '@polkadot/api-contract';
+import { ApiPromise } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+
+describe('Deploy flipper contract and test', () => {
```

Comment:
This must not be the name of your test. The name ought to explain what you are testing.

Reply:
Yes this is a copy-pasta error, good catch